### PR TITLE
Add docstrings in Beta elements in python_generator.cc

### PR DESCRIPTION
**Pull request by thunderboltsid**
Add docstrings mentioning that the Beta API is deprecated post 14.0.x in
- PrintBetaServicer
- PrintBetaStub
- PrintBetaServerFactory
- PrintBetaStubFactory

### CODE REVIEW

**Comment by thunderboltsid** [comment]
@nathanielmanistaatgoogle please take a look.

**Comment by sunwangme** [comment]
0.14.x？

File: src/compiler/python_generator.cc [hunk 1]
```diff
@@ -230,8 +230,10 @@ static void PrintAllComments(const DescriptorType* desc, Printer* printer) {
 
 bool PrintBetaServicer(const ServiceDescriptor* service, Printer* out) {
   out->Print("\n\n");
-  out->Print("class Beta$Service$Servicer(object):\n", "Service",
-             service->name());
+  out->Print("class Beta$Service$Servicer(object):\n"
+    "\"\"\"The Beta API is deprecated post the 14.0.x branch. It is "
```
Comment (nathanielmanistaatgoogle) [inline src/compiler/python_generator.cc]: Let's say "is deprecated for gRPC Python 0.15.0 and later". Can we also say something like "This [class, function] was generated only to ease transition from grpcio<0.15.0 to grpcio>=0.15.0"? Can we put in newlines so the generated text doesn't run off the (generated) page?

**Comment by nathanielmanistaatgoogle** [comment]
Let's do this against the `1.0.x` branch rather than `master`?

Looks good overall though.

**Comment by thunderboltsid** [comment]
This got too messy -_- Moving on to #8033

**Pull request closed without merging**